velox/exec/RowContainer.h

```diff
@@ -204,7 +204,8 @@ class RowContainer {
             false, // hasNext
             false, // isJoinBuild
             false, // hasProbedFlag
-            false, // hasNormalizedKey
+            false, // hasNormalizedKey,
+            false, // trackColumnsMayHaveNulls,
```

Comment:
@zhli1142015 Why do we need this flag? Any particular reason not to track this always?

Reply:
This change only affects the hash aggregation, hash join, and sort operators. I introduced this flag to ensure that other operators using RowContainer continue to behave as before. The operators not covered in this change include window, topN, topNRowNumber, and streaming aggregation. Please let me know if I should cover them in the same PR.

Reply:
Having extra flags adds complexity and makes it difficult to reason about the code. Unless there is a reason for some operators to not track nulls of the data stores in RowContainer let's track uniformly.

Reply:
Get it, this flag is removed, thanks.